samcli/commands/init/init_generator.py

```diff
@@ -4,11 +4,11 @@
 
 from samcli.commands.exceptions import UserException
 from samcli.lib.init import generate_project
-from samcli.lib.init.exceptions import GenerateProjectFailedError, ArbitraryProjectDownloadFailed
+from samcli.lib.init.exceptions import InitErrorException
 
 
 def do_generate(location, package_type, runtime, dependency_manager, output_dir, name, no_input, extra_context):
     try:
         generate_project(location, package_type, runtime, dependency_manager, output_dir, name, no_input, extra_context)
-    except (GenerateProjectFailedError, ArbitraryProjectDownloadFailed) as e:
```

Comment:
`InitErrorException` only has two subclasses, GenerateProjectFailedError, ArbitraryProjectDownloadFailed

Reply:
Is this related to the `InvalidLocationError`?

Reply:
`InvalidLocationError` inherents `InitErrorException` as well. So I intended to add `InvalidLocationError` to the list, which is equivalent to a single  `InitErrorException`